docs/examples/viz_animated_surfaces.py

```diff
@@ -0,0 +1,202 @@
+#          (default = 128)
+#
+
+time = 0
+dt = 0.1
+lower_xbound = -1
+upper_xbound = 1
+lower_ybound = -1
+upper_ybound = 1
+npoints = 128
+
+###############################################################################
+# creating the x, y points which will be used to fit the equation to get
+# elevation and generate the surface
+x = np.linspace(lower_xbound, upper_xbound, npoints)
+y = np.linspace(lower_ybound, upper_ybound, npoints)
+x, y = np.meshgrid(x, y)
+x = x.reshape(-1)
+y = y.reshape(-1)
+
+###############################################################################
+# Initializing wave_actors and adjusting lighting options to make the waves
+# look more aesthetically pleasant.
+
+
+def wave(x, y, equation, colormap_name):
+    xyz, colors = update_coordinates(x, y, equation=equation, time=time,
+                                     cmap_name=colormap_name)
+    wave_actor = actor.surface(xyz, colors=colors)
+    wave_actor.equation = equation
+    wave_actor.cmap_name = colormap_name
+    wave_actor.GetProperty().SetAmbient(1)
+    wave_actor.vertices = utils.vertices_from_actor(wave_actor)
+    wave_actor.no_vertices_per_point = len(wave_actor.vertices)/npoints**2
+    wave_actor.initial_vertices = wave_actor.vertices.copy() - \
+        np.repeat(xyz, wave_actor.no_vertices_per_point, axis=0)
+    return wave_actor
+
+
+###############################################################################
+# Equations to be plotted
+eq1 = "(np.sin(np.cos(3*np.pi*y)-time)*x+y*np.sin(np.cos(x*3*np.pi)-time))*\
+      0.48"
+eq2 = "0.72*((x**2 - y**2)/(x**2 + y**2))**(2)*np.cos(6*np.pi*x*y-time)/3"
+eq3 = "(np.sin(np.pi*2*x-np.sin(1.2*time))*np.cos(np.pi*2*y+np.cos(1.2*time)))\
+      *0.48"
+eq4 = "np.sin(2*np.pi*x**2+time)*np.cos(2*np.pi*y**2-time)*0.48"
+equations = [eq1, eq2, eq3, eq4]
+
+###############################################################################
+# List of colormaps to be used for the various functions.
+cmap_names = ['YlOrRd', 'plasma', 'viridis', 'Blues']
+
+###############################################################################
+# Creating a list of wave actors.
+waves = []
+for i in range(4):
+    waves.append(wave(x, y, equation=equations[i],
+                 colormap_name=cmap_names[i]))
+actors = waves
+
+
+###############################################################################
+# Creating a scene object and configuring the camera's position
+
+scene = window.Scene()
+scene.set_camera(position=(4.5, -15, 20), focal_point=(4.5, 0.0, 0.0),
+                 view_up=(0.0, 0.0, 1.0))
+showm = window.ShowManager(scene, size=(600, 600))
+
+###############################################################################
+# Creating an axes actor(for reference) and a grid to store multiple functions.
+# Adding them both to the scene.
+
+# To store the function names
+text = []
+for i in range(4):
+    t_actor = actor.text_3d("Function " + str(i+1), position=(-10, 0, 0),
```

Comment:
@SunTzunami here is a way you can make the text always follow the camera.

```python
t_actor = actor.label('Function ' + str(i + 1))
t_actor.SetCamera(scene.camera())
```

Although the combination of vtkFollower and vtkVectorText is not as powerful as the vtkTextActor3D and VTK already have the vtkBillboardTextActor3D, my suggestion is to use the label actor and create an issue asking for implementation of the BillboardText.

Reply:
Thanks for the snippet, implemented it.